pkg/virt-operator/resource/generate/components/validations_generated.go

```diff
@@ -9651,6 +9668,15 @@ var CRDsValidation map[string]string = map[string]string{
                         discovery. Defaults to false
                       type: boolean
                   type: object
+                pic:
+                  description: PIC enables/disables legacy Programmable Interrupt
+                    Controller.
+                  properties:
+                    enabled:
+                      description: Enabled determines if the feature should be enabled
+                        or disabled on the guest. Defaults to true.
```

Comment:
Could we revise this kind of description?  "Enabled determines if..." It seems not easy for me to understand.